package.json

```diff
@@ -15,14 +15,15 @@
     "grunt": "bin/grunt"
   },
   "dependencies": {
-    "findup-sync": "~0.3.0",
-    "grunt-known-options": "~1.1.0",
-    "nopt": "~3.0.6",
-    "resolve": "~1.1.0"
+    "grunt-known-options": "^1.1.0",
+    "interpret": "^1.1.0",
+    "liftoff": "^2.5.0",
+    "nopt": "^4.0.1",
+    "v8flags": "^3.0.1"
```

Comment:
are you sure you want to change from `~` to `^`?

Reply:
Fixed! Thanks!